tools/extras/install_mkl.sh

```diff
@@ -254,6 +254,16 @@ ConfigLdSo() {
     ldconfig )
 }
 
+(
+  set +u
```

Comment:
please clarify in a comment what this does

Reply:
done